dist/templates/ovnkube-alerts.yaml.j2

```diff
@@ -347,5 +347,15 @@ spec:
          {{ range query "ovn_controller_integration_bridge_geneve_ports_total{job='ovnkube-node', namespace='ovn-kubernetes'} !=  scalar(count(kube_node_info)) - 1" }}
          {{ .Labels.instance }}: {{ .Value }}
          {{ end }}.
- 
+
+    - alert: OVNKubeAllocatedSubnetsDoNotMatch
+      expr: |
+        ovnkube_clustermanager_allocated_v4_host_subnets{job="ovnkube-master", namespace="ovn-kubernetes"} !=
```

Comment:
why are you using this metric instead of the count?

Reply:
vnkube_clustermanager_allocated_v4_host_count metric tracks the total number of subnets possible for given CIDR, so that will be a constant value irrespective of the nodes allocated right? where as ovnkube_clustermanager_allocated_v4_host_subnets tracks the number of subnets allocated so we should be using this metric instead.

Reply:
ovnkube_clustermanager_allocated_v4_host_subnets metric tracks the total number of subnets possible for given network which will be a constant value irrespective of the nodes allocated right? where as ovnkube_clustermanager_allocated_v4_host_subnets tracks the number of subnets allocated so we should be using this metric instead.